frontend/feature/event/src/main/java/com/intersoft/event/EventManager.kt

```diff
@@ -36,6 +36,41 @@ object EventManager {
 
     }
 
+    fun editEvent(
+        eventId: Int,
+        eventName: String,
+        description: String,
+        selectedDateInMillis: Long,
+        durationInMillis:Long,
+        startTimeInMillis:Long,
+        maxNumberOfParticipants: Int,
+        location: String,
+        ownerId: Int,
```

Comment:
I would use a EventModel parameter here.

Reply:
EventModel is in :data:event, since the EventManager is being used in :app, this would break the planed system architecture